### PR TITLE
Update to gltf version 0.11

### PR DESCRIPTION
**Pull request by alteous**
Courtesy PR for updating to the new version of the gltf crate.

### CODE REVIEW

**Comment by alteous** [comment]
bors retry

File: Cargo.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ exclude = ["doc/*", "bors.toml", ".travis.yml", "test_data/*"]
 
 [features]
 default = ["opengl"]
-gltf-loader = ["gltf", "gltf-importer", "gltf-utils"]
+gltf-loader = ["gltf"]
```
Comment (kvark) [inline Cargo.toml]: let's remove this line entirely and just use `feature = "gltf"` everywhere

**Comment by alteous** [comment]
@kvark: :shipit:?

**Comment by kvark** [comment]
bors retry

**Comment by kvark** [comment]
Oh wow, the CI error seems legitimate and originates from cfg-if:
https://travis-ci.org/three-rs/three/jobs/394191323#L1962

**Comment by kvark** [comment]
Filed upstream in https://github.com/alexcrichton/cfg-if/issues/13

**Comment by kvark** [comment]
bors retry

**Pull request merged**